src/Doctrine/EventListener/PurgeHttpCacheListener.php

```diff
@@ -100,7 +105,11 @@ public function postFlush(): void
             return;
         }
 
-        $this->purger->purge(array_values($this->tags));
+        if ($this->invalidator instanceof PurgerInterface) {
+            $this->invalidator->purge(array_values($this->tags));
+        } elseif ($this->invalidator instanceof TagsInvalidatorInterface) {
```

Comment:
we do not want to introduce a new interface

Reply:
Why?